Source/Public/Set-VSTeamDefaultProject.ps1

```diff
@@ -49,19 +49,19 @@ function Set-VSTeamDefaultProject {
 
    process {
       if ($Force -or $pscmdlet.ShouldProcess($Project, "Set-VSTeamDefaultProject")) {
-         if (_isOnWindows) {
```

Comment:
can you explain in short the change in this cmdlet in regards to the process?

Reply:
The existing version sets an Environment variable "Team_Project" on windows but not on mac or linux.  The only part of this which should be in the _isOnWindows is persisting the value in the user or system environment, so the logic is now 
1. Set Team_project and Team_Process at process level on every OS.
2. If the OS is windows and $level is User or Machine , persist the variable at that level.